app/guid-node/metadata/-components/metadata-select/component.ts

```diff
@@ -1,23 +1,10 @@
 import Component from '@glimmer/component';
 import { inject as service } from '@ember/service';
 import Media from 'ember-responsive';
-import CedarMetadataTemplateModel from 'ember-osf-web/models/cedar-metadata-template';
-import { action } from '@ember/object';
 
-interface TabArgs {
-    template: CedarMetadataTemplateModel;
-}
-
-
-export default class MetadataSelect extends Component<TabArgs> {
+export default class MetadataSelect extends Component {
```

Comment:
May still want to specify the types of arguments passed to this component.

Reply:
What I am saying is that it is a good practice to specify the types of the arguments passed to this component. Are you saying that in your version the types exists on the file? In which case there might be some github magic happening and I'd suggest merging the feature branch into your local branch and push again.

Reply:
Still not seeing the args here.